### DISCUSSION TITLE
Custom Carrier Board - Display Port

### DISCUSSION THREAD

**Discussion by spb06** [OE4T/meta-tegra#1678] [Q&A] [answered]
Hi there, 

Thank you for all the support this repo/admins offer. 

### Background
We are working on bringing up a custom carrier board (Jetson AGX Orin) and have successfully gotten the serial USB port, ethernet ports, and CAN bus up and running (working off kirkstone). We are working on bringing up the display port now. 

### Problem
Even after porting the device tree, enabling certain drivers such as `TEGRA_DC` and `TEGRA_NVDISPLAY` amongst others display drivers via `menuconfig`, the display port does not come up. 

Do we have to do anything outside of this to enable the display port? The board vendor confirmed that they are not doing anything special to the display port and that it should be enabled by native NVIDIA drivers. 

I stumbled across this [this recipe](https://github.com/OE4T/meta-tegra/blob/kirkstone/recipes-kernel/nvidia-display-driver/nvidia-display-driver_35.5.0.bb). Do we need to use this recipe to enable the display port?

**Comment by ichergui** [comment]
Hi @spb06 
You need to make sure that you are using the right DTB

**Reply by spb06** [reply]
I believe I am using the right DTB since other peripherals are coming up fine. My gut feeling is that the recipe I linked above has something to do with fully enabling the Display Port. 

Does that recipe get included in my image for my custom machine by default?

**Reply by spb06** [reply]
Still trying to figure this one out, any help will be greatly appreciated. 

I flashed the orin devkit using via `tegra-demo-distro` for `demo-image-base` on `kirkstone` and the display port did not come up as well. However the serial and ethernet work just fine. Do I have to add any special display port configurations in my custom layer to enable the DP?

**Reply by spb06** [reply]
Also, could using `core-image-base` have something to do with the issues? Should I be using a graphics based image like `sato`?

**Reply by spb06** [reply, marked as answer]
Turns out the issue was using `core-image-base`. I used `core-image-x11` and `core-image-sato` and the display port came up fine. Thanks!

**Reply by dwalkes** [reply]
Thanks for updating @spb06 

I believe the intention was to present the terminal output on `demo-image-base` but I just confirmed I don't see that with my orin nano setup on kirkstone either.  I hadn't noticed this previously because I always just default to using SSH with this image.

**Reply by madisongh** [reply]
L4T R35.x doesn't have framebuffer console support for Orin. R36.x does, though.
